README.md

```diff
@@ -34,7 +34,7 @@
 </div>
 
 ## Overview
-OpenAPI Generator allows generation of API client libraries (SDK generation), server stubs,  documentation and configuration automatically given an [OpenAPI Spec](https://github.com/OAI/OpenAPI-Specification) (both 2.0 and 3.0 are supported). Currently, the following languages/frameworks are supported:
+OpenAPI Generator allows generation of API client libraries (SDK generation), server stubs,  documentation and configuration automatically given an [OpenAPI Spec](https://github.com/OAI/OpenAPI-Specification) (both 2.0 and . are supported). Currently, the following languages/frameworks are supported:
```

Comment:
@macjohnny looks like a search and replace erorr, and what about using `3.x` instead of `3.0` as newer version of OAS v3 has been released?

Reply:
I reverted this accidental change. I would leave it `3.0` instead of `3.x`, as it should be clear that major versions are supported, so I guess people looking for `3.0` as the main release can find it easily

Reply:
👌